app/web/features/search/MapWrapper.tsx

```diff
@@ -1,5 +1,6 @@
 import ReplayIcon from "@mui/icons-material/Replay";
 import TuneIcon from "@mui/icons-material/Tune";
+import { useMediaQuery, useTheme } from "@mui/material";
```

Comment:
Rather than `useTheme` you can just import our customized theme:

`import { theme } from "theme";`